src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -79,28 +80,30 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
 
 
         // --- mutation events bar chart ---
-        _.each(sequencedSamples, (sample: ClinicalDataBySampleId) => {
-            var _trackData = _.filter(this.props.mutations, function (_mutObj: any) {
-                return _mutObj.sampleId === sample.id;
-            });
-            tracksHelper.plotMuts(paper, config, chmInfo, rowIndex, _trackData, sample.id);
-            rowIndex = rowIndex + 1;
-
-            if (this.props.sampleManager.samples.length > 1) {
-                const id = `#mutTrack${sample.id}`;
-                const $container = $(id);
-                const pos = {x: parseInt($container.attr('x')) - 10, y: parseInt($container.attr('y')) - 5};
-                const $newContainer = $(`<svg id="${id}" height="12" width="12" />`);
-                $newContainer.attr(pos);
-                $container.replaceWith($newContainer);
-
-                let comp: any = this.props.sampleManager.getComponentForSample(sample.id);
-
-                ReactDOM.render(
-                    comp,
-                    $newContainer[0]
-                );
-            }
+        _.each(this.props.sampleManager.samples, (sample: ClinicalDataBySampleId) => {
+            if (mutSamples[sample.id]) {
```

Comment:
all thats changed is old line 82 replaced by new lines 83, 84, everything else is indentation change